src/include/uapi/drm_local/amdxdna_accel.h

```diff
@@ -13,6 +13,10 @@
 extern "C" {
 #endif
 
+#ifndef __counted_by
+#define __counted_by(cnt)
+#endif
+
```

Comment:
Instead of defining this, you may use #include <linux/stddef.h>.

Reply:
I thought about doing that, but at least on some of my machines __counted_by is not defined yet. Since it is still kind of new, support for it seems spotty.

Reply:
Which version of linux kernel are you running? I think this is supported in stddef.h after v6.6 kernel.

Reply:
This is the case, for example, in stock Ubuntu 22.04 where the kernel is 6.5.

Reply:
Did you patch the v6.5 kernel? xdna driver would not be compatible with standard v6.5 kernel.

Reply:
Yes, on my dev machine I have kernel 6.8. But when I run tests my Jenkins server doesn't have the latest kernel. We don't run any tests on our XDNA code yet, but we still need this to compile.

I'll modify this PR to include stddef.h and simply maintain this current hack in my own copy.

Reply:
Got it. Thanks for fixing this.